### ISSUE TITLE
Some content is not read by Talkback

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#550]
Samsung Galaxy J3 Prime
Android version 7.0

While Talkback is on, content "COVID Alert" receives focus; however it is not read by the screen reader on the COVID Alert is off screen.

Expected result
Talkback should announce it.

Screenshots
![menu1](https://user-images.githubusercontent.com/12256067/86966372-105b4380-c137-11ea-9759-9a27542dc41d.gif)

**Comment by JuliannaR** [comment]
@sarasuri I grouped this issue in https://github.com/cds-snc/covid-shield-mobile/issues/551
